### ISSUE TITLE
you should hide the api key

### ISSUE THREAD

**Issue by Setherizor** [NickJohn547745/DiscordBotSpeak#2]
wumpus is sad...

**Comment by Gibigbig** [comment]
came to comment this exact same thing

**Comment by LeightonBacon** [comment]
Yes, please remove this from the code or atleast make a fake one

**Comment by Gibigbig** [comment]
> The newer updates to the Discord API scour GitHub for key references and automatically generate a new one. Let's hope that it caught this and removed it.

they really shouldnt. I see this perpetuating bad practice.

**Comment by Frontear** [comment]
Honest mistakes are very real and possible. I argue this feature is a nice QOL thing to have, even _if_ it has the potential to perpetuate a bad practice. It's not like Discord advocates for it, this is a last-ditch effort to save the developer, Discord still actively promotes hiding the key.
